pkg/webhook/fleetresourcehandler/fleetresourcehandler_webhook_test.go

```
@@ -465,16 +613,28 @@ func TestHandleMemberCluster(t *testing.T) {
 		},
 	}
 
-	MCObjectBytes, err := json.Marshal(MCObject)
+	fleetMCObjectBytes, err := json.Marshal(fleetMCObject)
+	assert.Nil(t, err)
+	updateClusterResourceIDAnnotationFleetMCObjectBytes, err := json.Marshal(updateClusterResourceIDAnnotationFleetMCObject)
+	assert.Nil(t, err)
+	mcObjectBytes, err := json.Marshal(mcObject)
+	assert.Nil(t, err)
+	labelUpdatedFleetMCObjectBytes, err := json.Marshal(labelUpdatedFleetMCObject)
 	assert.Nil(t, err)
 	labelUpdatedMCObjectBytes, err := json.Marshal(labelUpdatedMCObject)
 	assert.Nil(t, err)
+	annotationUpdatedFleetMCObjectBytes, err := json.Marshal(annotationUpdatedFleetMCObject)
+	assert.Nil(t, err)
 	annotationUpdatedMCObjectBytes, err := json.Marshal(annotationUpdatedMCObject)
 	assert.Nil(t, err)
-	taintUpdatedMCObjectBytes, err := json.Marshal(taintUpdatedMCObject)
+	taintUpdatedFleetMCObjectBytes, err := json.Marshal(taintUpdatedFleetMCObject)
+	assert.Nil(t, err)
+	specUpdatedFleetMCObjectBytes, err := json.Marshal(specUpdatedFleetMCObject)
 	assert.Nil(t, err)
 	specUpdatedMCObjectBytes, err := json.Marshal(specUpdatedMCObject)
 	assert.Nil(t, err)
+	statusUpdatedFleetMCObjectBytes, err := json.Marshal(statusUpdatedFleetMCObject)
+	assert.Nil(t, err)
 	statusUpdatedMCObjectBytes, err := json.Marshal(statusUpdatedMCObject)
 	assert.Nil(t, err)
```

nit: just ignore the err?